vision/standard_object/odlc_contour_filtering.py

```diff
@@ -302,15 +307,19 @@ def test_circleness(img: consts.ScImage) -> bool:
     -------
     is_circular : bool
         Returns true if circle of appropriate size is found (to reduce chance of false positives)
+
+    References
+    ----------
+    Blur kernel and sigmaX cv2.GaussianBlur() parameters recommended by opencv documentation
+    see: ( i couldnt fit it on one line with the https:// :[ )
+    docs.opencv.org/4.x/dd/d1a/group__imgproc__feature.html#ga47849c3be0d0406ad3ca45db65a25d2d
     """
     padding: int = int(img.shape[0] * 0.05)
     modded: consts.ScImage = cv2.copyMakeBorder(
         img, padding, padding, padding, padding, cv2.BORDER_CONSTANT, None, 0
     )
-    # 5 chosen arbitrarily as the kernel size and sigmaX params for the blur function
-    # (kernel size and sigmaX are both parameters of cv2.GaussianBlur())
-    # the number should not be very important because the input image is a binarized image
-    modded = cv2.GaussianBlur(img, (5, 5), 5)
+
+    modded = cv2.GaussianBlur(img, (7, 7), 1.5)
```

Comment:
might make this more clear
```suggestion

    modded = cv2.GaussianBlur(img, ksize=(7, 7), sigmaX=1.5)
```